app/models/hackbot/conversations/check_in.rb

```diff
@@ -143,6 +151,10 @@ def wait_for_notes(event)
 
       private
 
+      def record_notes(event)
+        data['notes'] = event[:text] unless event[:text] =~ /^(no|nope|nah)$/i
```

Comment:
@zachlatta: What about `save_notes`?

It could behave similar to `ActiveRecord.save`, in the sense it would return a boolean depending on whether or not the saving actually occurred.

Another option is `process_notes`, but I think that sounds a bit too middleware-y.